Minor edits and bug fixes

rdotanirudh opened the pull request:
1) parameters.ini file rewritten to show how it would actually look after run.sh execution.
2) create_basic_characteristics_table.py corrected to get Rcloud in the correct units.
3) slick_run.py edited to use the SLURM_ARRAY_TASK_ID properly.
4) slick_new.sh edited to make the initial basic table creation jobscript use only 1 core with 12 GB to use space efficiently.

rdotanirudh committed:
Update slick_run.py

Fixed a minor issue with the array task IDs being read as enumerate starts indexing from 0 while SLURM_ARRAY_TASK_ID starts from 1.

rdotanirudh committed:
Fixed units issue for Rcloud

rdotanirudh committed:
Update slick_new.sh

Adding sbatch and run parameters for parameters.ini and changing run.sh script to only use 1 CPU with larger memory as creating the basic table is not parallelized but needs a lot of memory to load and use the snapshot.

rdotanirudh committed:
Update parameters.ini

Showing how the parameters.ini file would look once run.sh is executed in both total and randomize modes.

Haogy101 commented:
Changes 1, 3, and 4 look good. I can't say about change 3 though since I didn't write that originally.

smsutherland commented:
Changes 1, 3, and 4 look good. I don't know about change 2 since I didn't write that part of the code.

rdotanirudh committed:
Update create_cloudspercore_list.py

rdotanirudh committed:
Update slick_run.py

Removing the necessity of having only clouds from the same galaxy on a line in the Clouds_per_core file.

rdotanirudh committed:
Major update to limfunctions

Added new DMR calculation, CO areal integrated source brightness, HI and H+ fraction, and minor edits. Also updated slick to show densities for unconverged clouds as well by introducing a new function compute_temp_dens.

rdotanirudh committed:
Major update

This version now allows computation of RadField by estimating it from local SFR surface density (using a 64 neighbour KDTree), using SIMBA dust masses to find optical depth and attenuate the RadField based on Draine's extinction law, and using Qi Li's paper to find DMR for each cloud. Also contains minor bug fixes.

rdotanirudh committed:
Minor edits and removing unnecessary time limits
